`test_create_link` needs to wait

Sometimes, `test_create_link` manages to load the deeply-nested element we use to check if the playback succeeded too quickly, and we fail with:
```
AssertionError: assert 'Example Domain' in ''
```
Let's tweak so that this test is less flaky. A sleep? Or, maybe, just making the selector more detailed, so that Playwright's built-in waiting mechanism works even better: perhaps we wait for the `h1`?

This should be quick to fix.

--- a/perma_web/functional_tests/test_links.py
+++ b/perma_web/functional_tests/test_links.py
@@ -22,7 +22,7 @@ def test_create_link(logged_in_user) -> None:
             .frame_locator('.archive-iframe') \
             .frame_locator('iframe') \
             .frame_locator('iframe') \
-            .locator('html').inner_text()
+            .locator('h1').inner_text()
 
 
 def test_link_required(logged_in_user) -> None:

